src/BlazorUI/Bit.BlazorUI/Components/Utilities/Element/BitElement.scss

```
@@ -0,0 +1,15 @@
+﻿@import "../../../Styles/functions.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)